GitUI/UserControls/FilterToolBar.cs

```diff
@@ -305,6 +282,31 @@ private void UpdateBranchFilterItems()
                 await this.SwitchToMainThreadAsync();
                 BindBranches(branches);
             }).FileAndForget();
+
+            return;
+
+            void BindBranches(string[] branches)
```

Comment:
```suggestion
            private void BindBranches(string[] branches)
```

Reply:
This is a local method always private to the enclosing method